### PR TITLE
Conference Information from ACCU, Cpp on Sea and London Cpp Meetup

### PR DESCRIPTION
**Pull request by hniemeyer**
Add descriptions and slides for my talks at ACCU, Cpp On Sea and the London C++ Meetup. The README now also links to these newly created folders.

### CODE REVIEW

File: ACCU/2022/what-has-cpp20-ever-done-for-templates.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# What Has C++20 Ever Done For Templates?
+
+## Author
+
+Hendrik is a C++ developer and works on software for analysis of pipeline inspection results. This includes topics like machine learning, numerical mathematics and distributed computing. Before this he completed his PhD in physics at the University of Osnabrück with a thesis about quantum mechanics and + numerical simulations where he got to know and and love programming and complex, mathematical tasks. His favorite programming languages, in which he also has the most experience, are C++ and Rust. He describes himself as a "learning enthusiast"
```
Comment (wagnerpeer) [inline ACCU/2022/what-has-cpp20-ever-done-for-templates.md]:
```suggestion
Hendrik is a C++ developer and works on software for analysis of pipeline inspection results. This includes topics like machine learning, numerical mathematics and distributed computing. Before this he completed his PhD in physics at the University of Osnabrück with a thesis about quantum mechanics and + numerical simulations where he got to know and love programming and complex, mathematical tasks. His favorite programming languages, in which he also has the most experience, are C++ and Rust. He describes himself as a "learning enthusiast"
```

File: ACCU/2022/a-friendly-introduction-to-rust-for-cpp-developers.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# A Friendly Introduction to Rust for C++ Developers
+
+## Author
+
+Hendrik is a C++ developer and works on software for analysis of pipeline inspection results. This includes topics like machine learning, numerical mathematics and distributed computing. Before this he completed his PhD in physics at the University of Osnabrück with a thesis about quantum mechanics and + numerical simulations where he got to know and and love programming and complex, mathematical tasks. His favorite programming languages, in which he also has the most experience, are C++ and Rust. He describes himself as a "learning enthusiast"
```
Comment (wagnerpeer) [inline ACCU/2022/a-friendly-introduction-to-rust-for-cpp-developers.md]:
```suggestion
Hendrik is a C++ developer and works on software for analysis of pipeline inspection results. This includes topics like machine learning, numerical mathematics and distributed computing. Before this he completed his PhD in physics at the University of Osnabrück with a thesis about quantum mechanics and + numerical simulations where he got to know and love programming and complex, mathematical tasks. His favorite programming languages, in which he also has the most experience, are C++ and Rust. He describes himself as a "learning enthusiast"
```

File: ACCU/2022/what-has-cpp20-ever-done-for-templates.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+who always gets absorbed in trying out new things. Therefore, he values being up to date with programming languages
+and using the latest features of them in a meaningful way.
+
+[Twitter Profile](https://twitter.com/hniemeye)
+
+## Abstract
+
+The answer is a lot! There are some smaller features like more possibilities for non-type template parameters
+(literal class types and floating points are now possible) and lambda expressions now allow template type parameters in their definition.
+But the biggest feature is clearly concepts which allow us to easily define requirements on template arguments which are enforced by the compiler.
```
Comment (wagnerpeer) [inline ACCU/2022/what-has-cpp20-ever-done-for-templates.md]:
```suggestion
But the biggest feature is clearly concepts, which allow us to easily define requirements on template arguments, which are enforced by the compiler.
```

File: CppOnSea/2020/intro-to-cpp20-concepts.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Hendrik is a Technical Lead for Data Sciece and works on machine learning and big data solutions in Python but is also interested in C++ and Rust. He described himself as a "learning enthusiast" who always gets absorbed in learning new things.
+
+[Twitter Profile](https://twitter.com/hniemeye)
+
+## Abstract
+
+One of the biggest new features of C++20 are concepts. Concepts define requirements on template arguments which are enforced by the compiler. This leads to more readable code which clearly expresses intent. Error messages are improved since violations of the constraints defined by concepts can be detected in the early stages of the template instantiation.
+
+This talked is aimed at people with little or no experience with concepts and will introduce generic programming with concepts (constraining templates, writing your own concepts and overloading functions which are constrained by concepts).
```
Comment (wagnerpeer) [inline CppOnSea/2020/intro-to-cpp20-concepts.md]:
```suggestion
This talk is aimed at people with little or no experience with concepts and will introduce generic programming with concepts (constraining templates, writing your own concepts and overloading functions which are constrained by concepts).
```

File: ACCU/2021/a-practical-introduction-to-cpp-modules.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+Modularity is a long known principle for creating maintainable, changeable and durable software. In contrast to other programming languages, C++ did not have its own module system and used header files to structure code. But this has changed with C++20 which introduced modules as one of its bigger new features.
+
+In this talk I will discuss the theoretical side of modules and their advantages over header files (e.g. faster compile times, invisibility of macros, preprocessor directives and non-exported entities, fewer name collisions and more) but the main focus will be the practical usage of modules today.
+
+I will demonstrate how modules can be used today with the three major compilers and how to restructure your projects using modules. Also the support of build systems for modules will be discussed.
+
+## Video
+
+[A Practical Introduction to C++20's Modules](https://youtu.be/tOjZtodXqNQ)
```
Comment (wagnerpeer) [inline ACCU/2021/a-practical-introduction-to-cpp-modules.md]:
```suggestion
[![A Practical Introduction to C++20's Modules](https://img.youtube.com/vi/tOjZtodXqNQ/0.jpg)](https://youtu.be/tOjZtodXqNQ)
```

File: ACCU/2022/a-friendly-introduction-to-rust-for-cpp-developers.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+## Abstract
+
+Rust’s design philosophy aims for a language which helps the developer as much as possible with memory safety, freedom from data races, great error messages and package management. But also no sacrifices shall be made in terms of performance and low-level control over memory.
+
+In this talk, I will introduce Rust and its core principles from the perspective of a C developer. Similarities and differences between C++ and Rust will be discussed by comparing the example code in both languages.
+
+## Video
+
+[A Friendly Introduction to Rust for C++ Developers](https://youtu.be/o-k3xgV0oBE)
```
Comment (wagnerpeer) [inline ACCU/2022/a-friendly-introduction-to-rust-for-cpp-developers.md]:
```suggestion
[![A Friendly Introduction to Rust for C++ Developers](https://img.youtube.com/vi/o-k3xgV0oBE/0.jpg)](https://youtu.be/o-k3xgV0oBE)
```

File: ACCU/2022/what-has-cpp20-ever-done-for-templates.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+This talk will give a tour of the new features of templates in C++20 and will discuss how the new features work and what they can be used for.
+
+## Video
+
+[What Has C++20 Ever Done For Templates?](https://youtu.be/Q43YyGQXHxg)
```
Comment (wagnerpeer) [inline ACCU/2022/what-has-cpp20-ever-done-for-templates.md]:
```suggestion
[![What Has C++20 Ever Done For Templates?](https://img.youtube.com/vi/Q43YyGQXHxg/0.jpg)](https://youtu.be/Q43YyGQXHxg)
```

File: CppOnSea/2020/intro-to-cpp20-concepts.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+## Video
+
+[An Introduction to C++20’s Concepts](https://youtu.be/N_kPd2OK1L8)
```
Comment (wagnerpeer) [inline CppOnSea/2020/intro-to-cpp20-concepts.md]:
```suggestion
[![An Introduction to C++20’s Concepts](https://img.youtube.com/vi/N_kPd2OK1L8/0.jpg)](https://youtu.be/N_kPd2OK1L8)
```

File: CppOnSea/2022/what-has-cpp20-ever-done-for-templates.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+## Abstract
+
+The answer is a lot! There are some smaller features like more possibilities for non-type template parameters (literal class types and floating points are now possible) and lambda expressions now allow template type parameters in their definition. But the biggest feature is clearly concepts which allow us to easily define requirements on template arguments which are enforced by the compiler.
+
+This talk will give a tour of the new features of templates in C++20 and will discuss how the new features work and what they can be used for.
+
+## Video
+
+[What Has C++20 Ever Done For Templates?](https://youtu.be/KRx8qQXm0fc)
```
Comment (wagnerpeer) [inline CppOnSea/2022/what-has-cpp20-ever-done-for-templates.md]:
```suggestion
[![What Has C++20 Ever Done For Templates?](https://img.youtube.com/vi/KRx8qQXm0fc/0.jpg)](https://youtu.be/KRx8qQXm0fc)
```

File: LondonCppMeetup/2020/a-friendly-introduction-to-rust.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+## Abstract
+
+Rust's design philosophy aims for a language which helps the developer as much as possible with memory safety, freedom from data races, great error messages and package management. But also no sacrifices shall be made in terms of performance and low-level control over memory.
+
+In this talk, I will introduce Rust and its core principles from the perspective of a C++ developer. Similarities and differences between C++ and Rust will be discussed by comparing the example code in both languages.
+
+## Video
+
+[A Friendly Introduction to Rust](https://youtu.be/NpBk_H7S3zM)
```
Comment (wagnerpeer) [inline LondonCppMeetup/2020/a-friendly-introduction-to-rust.md]:
```suggestion
[![A Friendly Introduction to Rust](https://img.youtube.com/vi/NpBk_H7S3zM/0.jpg)](https://youtu.be/NpBk_H7S3zM)
```

**Review by wagnerpeer — CHANGES_REQUESTED** [review]
Please update.

**Comment by hniemeyer** [comment]
accepted your changes.

**Review by wagnerpeer — APPROVED** [review]

**Pull request merged**